rust/src/mqtt/mqtt_property.rs

```diff
@@ -24,7 +24,7 @@ use nom7::IResult;
 
 // TODO: It might be useful to also add detection on property presence and
 // content, e.g. mqtt.property: AUTHENTICATION_METHOD.
-#[derive(Debug)]
+#[derive(Debug, PartialEq, PartialOrd, Eq, Ord)]
```

Comment:
Are `Eq` and `Ord ` required for the ops that need to be done with this enum?

Reply:
I was getting errors while the comparison assertion. Do you want me to remove these?

Reply:
I do not see any errors after removing these or their usage. You'd need `PartialEq` and `PartialOrd` indeed. Could you please check again? Let me know if I'm doing something wrong..

Reply:
Ok I will update and create a new PR.